### PR TITLE
STONEBLD-1936: Added retry mechanism and don't fail the pipeline on show-sbom task failure

### PR DESCRIPTION
**Pull request by tisutisu**
Addresses: https://issues.redhat.com/browse/STONEBLD-1936
# Before you complete this pull request ...

Look for any open pull requests in the repository with the title "e2e-tests update" and 
see if there are recent e2e-tests updates that will be applicable to your change.

### CODE REVIEW

File: task/show-sbom/0.1/show-sbom.yaml [hunk 1]
```diff
@@ -25,11 +25,20 @@ spec:
...
-         cat err >&2
-       fi
-       exit $RET
+      #!/busybox/sh
+      status=-1
+      max_try=5
+      wait_sec=2
+      for run in $(seq 1 $max_try); do
+        status=0
+        cosign download sbom $IMAGE_URL 2>err
```
Comment (chmeliik) [inline task/show-sbom/0.1/show-sbom.yaml:34]: Do we want to print only the last error? We could collect all the errors with `2>>err`

**Review by chmeliik — COMMENTED** [review]
> STONEBLD-1936 Fix 

Could you improve the commit message?

File: task/show-sbom/0.1/show-sbom.yaml (hunk 1 above)
Comment (tisutisu) [reply task/show-sbom/0.1/show-sbom.yaml]: Good point. Updated.

**Review by chmeliik — APPROVED** [review]
LGTM :+1:

**Pull request merged**